CMakeLists.txt

```diff
@@ -338,6 +338,8 @@ if (UNIX AND NOT APPLE)
   find_package(Threads)
 endif()
 
+add_definitions(-DAUTO_INITIALIZE_EASYLOGGINGPP)
```

Comment:
Sure. Debug build (i.e. shared libs build) for libeasylogging requires the symbol `el::base::elStorage` (aka `ELPP`) to be found when building, which is only defined in mlog.cpp in libepee (which depends on libeasylogging) through the macro `INITIALIZE_EASYLOGGINGPP`. By removing this macro in mlog.cpp and defining `AUTO_INITIALIZE_EASYLOGGINGPP`, that symbol gets defined automatically in easylogging++.cc, hence libeasylogging now builds fine.